libs/common-services/src/main/java/uk/gov/di/ipv/core/library/persistence/item/IpvSessionItem.java

```diff
@@ -36,6 +36,7 @@ public class IpvSessionItem implements PersistenceItem {
     private String emailAddress;
     private ReverificationStatus reverificationStatus;
     private List<String> stateStack = new ArrayList<>();
+    private String journeyContext;
```

Comment:
The main thing I want to avoid if we introduce this is having fiddly bits of state that are hidden from the journey map and end up being missed. We've had several of bugs caused by a bit of state being set, and then becoming invalid - e.g. we set the VOT in CheckExistingIdentity, but forgot to reset it when the user asks to update their details.

Some things to consider:
- is it obvious what 'journey context' means?
  - does it need an explanatory comment?
- when does the journey context get reset?
  - If it should only be for things that apply to an entire journey, do we need to say so?
  - If that's too restrictive, we need to be very careful to identify when it no longer applies
- might we ever want to set two overlapping journey context values?
  - Should it be multivalued, and if so, how would that work - a list or a map?
  - (This might just add a bunch of complexity for no reason)

The `rfc` usage at the moment seems relatively risk-free at the moment... either a journey needs to perform a repeat fraud check or not, even if you decide to change your name. 🙂

Reply:
- I think it should probably have some kind of comment in this file - but we could also refer to it in the team manual if there's an appropriate place.

- Yeah, we can worry about that later - probably either distinguishing null vs. empty, or using a magic 'delete' value

- I think the complexity with a list is knowing when to append vs replace... it's fine to leave as a string for now :)